src/lib.rs

```diff
@@ -67,6 +67,10 @@ pub struct Detection {
     /// If null, means the system was not able to identify the records present along with this
     /// object with any specific UAS.
     pub uas_id: Option<u64>,
+    /// If this detection set refers to a UAV and if applicable, this member refers to the "home location" set on the
```

Comment:
For detection, this should say "detection record" rather than "detection set", as different records share in theory no correlation

Reply:
That is wrong and they do in fact share the UAS ID so they do refer to the same UAS (as long as the field is filled in)

Reply:
I understand. Maybe it should be more explicit that the detection set refers to the same entity by including specific UAS in the description?